lib/pp.mli

```diff
@@ -10,30 +10,30 @@
-(* they should instead rely on the available functions below.           *)
-(*                                                                      *)
-(* Box order and number is indeed an important factor. Try to create    *)
-(* a proper amount of boxes. The `++` operator provides "efficient"     *)
-(* concatenation, but using the list constructors is usually preferred. *)
-(*                                                                      *)
-(* That is to say, this:                                                *)
-(*                                                                      *)
-(* `hov [str "Term"; hov (pr_term t); str "is defined"]`                *)
-(*                                                                      *)
-(* is preferred to:                                                     *)
-(*                                                                      *)
-(* `hov (str "Term" ++ hov (pr_term t) ++ str "is defined")`            *)
-(*                                                                      *)
-(************************************************************************)
+(** Pretty printing guidelines *******************************************)
+(**                                                                      *)
+(** `Pp.t` is the main pretty printing document type                     *)
+(** in the Coq system. Documents are composed laying out boxes, and      *)
+(** users can add arbitrary tag metadata that backends are free          *)
+(** to interpret.                                                        *)
+(**                                                                      *)
+(** The datatype has a public view to allow serialization or advanced    *)
+(** uses, however regular users are _strongly_ warned against its use,   *)
+(** they should instead rely on the available functions below.           *)
+(**                                                                      *)
+(** Box order and number is indeed an important factor. Try to create    *)
+(** a proper amount of boxes. The `++` operator provides "efficient"     *)
+(** concatenation, but using the list constructors is usually preferred. *)
+(**                                                                      *)
+(** That is to say, this:                                                *)
+(**                                                                      *)
+(** `hov [str "Term"; hov (pr_term t); str "is defined"]`                *)
+(**                                                                      *)
+(** is preferred to:                                                     *)
+(**                                                                      *)
+(** `hov (str "Term" ++ hov (pr_term t) ++ str "is defined")`            *)
+(**                                                                      *)
+(*************************************************************************)
```

Comment:
Alternatively we could just drop all the per-line `(**` and use a single `(** ... *)` block; we also need to update the syntax from "markdown" to "ocamldoc".

Reply:
I don't know ocamldoc syntax

Reply:
Mainly we need to turn quotes into brackets, see https://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#ss:ocamldoc-formatting